Documentation: Doc/several operators page (#646)
pyansys/pydpf-core: docs/source/operator_reference.rst

Operators
=========

> DPF operators allow the manipulation and the transformation of simulation data.

Reviewer: ```suggestion
DPF operators are used to manipulate and transform simulation data.
```

Reviewer: @PProfizi You can use provide in the way written--or change it to:
DPF operators are used to manipulate and transform simulation data.

> From DPF Server for Ansys 2023 R2 onwards, the operators available in DPF will depend on the active

Reviewer: ```suggestion
From DPF Server for Ansys 2023 R2 and later, the operators available in DPF depend on the active
```

`ServerContext <https://dpf.docs.pyansys.com/api/ansys.dpf.core.server_context.html#servercontext>`_.

The available contexts are **Entry** and **Premium**. Click a section to discover the operators available in your context. 

> For more information about **Entry** and **Premium** contexts use, see :ref:`user_guide_server_context`.

Reviewer: ```suggestion
For more information about using these two contexts, see :ref:`user_guide_server_context`.
```

.. grid:: 2

   .. grid-item::
        .. card:: Operators (Entry)
            :img-top: _static/assets/basic.png
            :link-type: doc
            :link: operator_reference_entry

            Click here to get started with operators available in DPF Entry.

            +++
            .. button-link:: ENTRY
               :color: secondary
               :expand:
               :outline:
               :click-parent:              

   .. grid-item::
        .. card:: Operators (Premium)
            :img-top: _static/assets/premium.png
            :link-type: doc
            :link: operator_reference_premium

            Click here to get started with operators
            available in DPF Premium.

            +++
            .. button-link:: PREMIUM
               :color: secondary
               :expand:
               :outline:
               :click-parent:

.. note::
    For Ansys 2023 R1 and older, the Premium documentation should be used.

>     Depending on the server version,  some operators in the documentation may 
>     not be available.

Reviewer: ```suggestion
    Some operators in the documentation might not be available for a particular server version.
```
